### PR TITLE
[FEATURE] Added index pattern fields refreshing from events

### LINKED ISSUE #2630: Improved management of index-pattern fields
### Description
Currently, the fields of the index-pattern used in the plugin are updated when the "Health-Check" is executed and when a visualization fails.

This management has to be improved to avoid showing a field that is not in the index-pattern.

### Tasks

- [ ] Add new fields to the template
- [ ] Create an observer in the event view to preventing "warnings".

### PR DESCRIPTION
**Pull request by frankeros**
Hi team! This resolves:

- Refresh index pattern fields when there are some unknown fields in the document viewer from Events.

Closes #2630

### CODE REVIEW

**Comment by Desvelao** [comment]
I tested the PR, and it works.

The toast:
image

And the field shows a `?` icon,  but after reloading the page, this disappears
image

**Review by Desvelao — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 ### Added
 
+- Improved management of index-pattern fields [#2630] (https://github.com/wazuh/wazuh-kibana-app/issues/2630)
```
Comment (jsanchez91) [inline CHANGELOG.md:9]: Update the branch and move this to the 4014 revision

**Review by jsanchez91 — CHANGES_REQUESTED** [review]

**Review by jsanchez91 — APPROVED** [review]
LGTM!

**Pull request merged**